pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -195,13 +195,26 @@
         "mkdir -p /tmp/.goss-dir"
       ]
     },
+    {
+      "type": "shell",
+      "inline": [
+      "mkdir -p /tmp/bin/"
+      ]
+    },
     {
         "type": "file",
         "source": "/usr/local/bin/goss",
         "destination": "/tmp/.goss-dir/goss",
         "direction": "upload",
         "max_retries": 10
     },
+    {
```

Comment:
what do you think to upload the `mindthdgap` binary in ansible? you can add retries and remove the binary once upload is finished. 
I was planning to try the same for `goss` binary.

Reply:
I don't get what the benefit would be of using ansible. Packer accomplishes the job for us

Reply:
I am thinking 
- ansible playbook is common for any providers so code change stays at one place.
- the source binary dont have to be at the `/usr/local/bin` . however both packer and ansible can use variable to represent source path. .i.e. ansible playbook does not need to assume that binary exists at `/tmp/bin/mindthegap` 
- remove `mindthegap` binary after ansible task is finished as we dont want to leave it behind in the AMI.

Reply:
We need to keep the `mindthegap` binary for pre-provisioned deployments. 

> the source binary dont have to be at the /usr/local/bin
this is where the mindthegap lives on the devkit container and production container. If we make an `arm` version we will still need to package both.

> ansible playbook is common for any providers so code change stays at one place.

this is the biggest selling point to me